content/docs/command-reference/plots/show.md

```diff
@@ -209,12 +208,12 @@ In many automation scenarios (like
 output the [Vega-Lite](https://vega.github.io/vega-lite/) spec file instead of
 rendering an HTML plot. For example, to generating another image format like PNG
 or JPEG, or to include it differently into a web/mobile app. The `--show-vega`
-option prevents wrapping this plot spec in HTML. Note that the resulting file is
-JSON:
+option prevents wrapping this plot spec in HTML. Note that the command will
+print the JSON in the terminal:
 
 ```dvc
-$ dvc plots show --show-vega logs.csv -y accuracy
-file:///Users/usr/src/plots/logs.csv.json
+$ dvc plots show --show-vega logs.csv -y accuracy > logs.csv.json
+$ cat logs.csv.json
 ```
 
 ```json
```

Comment:
This is fine but we could simplify to just:
````suggestion
$ dvc plots show --show-vega logs.csv -y accuracy
```

```json
````
Since the JSON is shown right under it (and to avoid `cat` which is Linux-specific). Or even merge the JSON block into the console sample. WDYT?